test/cpp/end2end/xds/xds_end2end_test.cc

```diff
@@ -6071,11 +6090,7 @@ class XdsSecurityTest : public XdsEnd2endTest {
     balancer_->ads_service()->SetEdsResource(BuildEdsResource(args));
   }
 
-  void TearDown() override {
-    g_fake1_cert_data_map = nullptr;
-    g_fake2_cert_data_map = nullptr;
-    XdsEnd2endTest::TearDown();
-  }
+  void TearDown() override { XdsEnd2endTest::TearDown(); }
```

Comment:
This override isn't needed at all anymore if all it's going to do is delegate to the base class.